app/controllers/carl_people_controller.rb

```diff
@@ -0,0 +1,47 @@
+class CarlPeopleController < ApplicationController
+  #----------------- SHOW A PERSON --------------------
+  def show
+    @person = Person.find(params[:id])
+    @tasks_count = @person.tasks.count
+    @tasks_wip = @person.tasks.where(date_complete: nil).count
+    @tasks_done = @tasks_count - @tasks_wip
+  rescue
+    redirect_to "/person_not_found"
+  end
+
+  #----------------- SHOW A PERSON'S TASKS --------------------
+  def tasks
+    @person = Person.find(params[:id])
+    @tasks = Task.all.where(person_id: params[:id])
+
+    @people = [{
+        id: @person.id,
+        name: @person.name,
+        count: @person.tasks.where(date_complete: nil).count
+      }]
```

Comment:
If you make a method on `Person` for `.tasks.where(date_complete: nil).count` (maybe `incomplete_task_count`?) you could just pass through the `@person` instead of creating this hash. That gives you a more flexible and powerful interface to work with and allows you to move logic related to the `Person` model out of the controller and into the `Person` model where it is more easily shared between controllers.